test/support/DesData.scala

```diff
@@ -38,8 +38,10 @@ object DesData {
 
   val directDebitData: DirectDebitData = DirectDebitData(Some(List(DirectDebitDetails("Tester Surname", "404784", "70872490"))))
 
+  private val items: Seq[Item] = Seq(Item(Some(LocalDate.parse("2018-03-31"))))
+
   val financialData: FinancialData =
-    FinancialData("VRN", vrn.value, "VATC", "2019-08-20T10:44:05Z", Seq(Transaction("VAT Return Credit Charge", Option("18AC"))))
+    FinancialData("VRN", vrn.value, "VATC", "2019-08-20T10:44:05Z", Seq(Transaction("VAT Return Credit Charge", Option("18AC"), Option(items))))
```

Comment:
will the new items appear in all responses or is there a way to control whether it's there or not for testing purposes?

Reply:
far as I can tell they'll always be there now, not sure if we can but that would be helpful!
I think even if we have an empty one in there from the des response we get an empty items object

Reply:
ah wait, hold on - for some reason I thought this was the stubs and this was the data being returned by the stub. Is there a way to control the clearing date in the stub for testing purposes?